modules/apps/transfer/keeper/forwarding.go

```diff
@@ -19,8 +19,8 @@ func (k Keeper) forwardPacket(ctx sdk.Context, data types.FungibleTokenPacketDat
 		nextForwardingPath = types.NewForwarding(false, data.Forwarding.Hops[1:]...)
 	}
 
-	// sending from the forward escrow address to the original receiver address.
-	sender := types.GetForwardAddress(packet.DestinationPort, packet.DestinationChannel)
+	// sending from module account (used as a temporary forward escrow) to the original receiver address.
+	sender := k.authKeeper.GetModuleAddress(types.ModuleName)
```

Comment:
looks like `k.authKeeper.GetModuleAddress(types.ModuleName)` is already a map lookup, are you thinking of something like `k.GetModuleAddress()` as a utility to delegate to the authKeeper?

Reply:
happy to have it called directly in the utility fn!